rosbag2_performance/rosbag2_benchmarking/rosbag2_benchmarking/raport_gen.py

```diff
@@ -0,0 +1,434 @@
+# Copyright 2020, Robotec.ai sp. z o.o.
```

Comment:
should that file be renamed to `report`?

Reply:
Yes, it should. Will fix it, thanks :smiley: